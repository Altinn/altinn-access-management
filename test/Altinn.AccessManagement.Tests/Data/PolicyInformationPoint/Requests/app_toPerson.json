{
  "subject": {
    "id": "urn:altinn:userid",
    "value": "20001337"
  },
  "party": {
    "id": "urn:altinn:partyid",
    "value": "50001337"
  },
  "resource": [
    {
      "id": "urn:altinn:org",
      "value": "org1"
    },
    {
      "id": "urn:altinn:app",
      "value": "app1"
    }
  ]
}