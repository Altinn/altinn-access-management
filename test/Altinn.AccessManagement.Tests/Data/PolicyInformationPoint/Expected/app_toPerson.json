[
  {
    "delegationchangeid": 1337,
    "resourceregistrydelegationchangeid": 0,
    "delegationchangetype": 1,
    "resourceid": "org1/app1",
    "resourcetype": "AltinnAppId",
    "offeredbypartyid": 50001337,
    "coveredbypartyid": null,
    "coveredbyuserid": 20001337,
    "performedbyuserid": 20001336,
    "performedbypartyid": null,
    "blobstoragepolicypath": "org1/app1/50001337/u20001337/delegationpolicy.xml",
    "blobstorageversionid": "CorrectLeaseId",
    "created": "2022-11-29T10:19:45"
  }
]